distribution/lib/Standard/Base/0.0.0-dev/src/Function.enso

```diff
@@ -75,7 +78,7 @@ identity x = x
    > Example
     IO.println <| Function.flip (+) "world" "hello" # Prints 'helloworld'
 
-# flip : (a -> b -> c) -> (b -> a -> c)
+   ## flip : (a -> b -> c) -> (b -> a -> c)
 flip : (Any -> Any -> Any) -> (Any -> Any -> Any)
 flip f = (x -> y -> f y x)
```

Comment:
Why are all of these indented like this?

Reply:
It separated the doc comment from the function and I don't trust something bad wont happen!

Reply:
Ah, now I see. Yeah it seems sketchy.

I think we should remove this 'fake' signatures altogether. We can readd them once they are valid. They are well-known so not like we will forget what to put there. What do you think?